content/source/docs/cloud/api/changelog.html.md

```diff
@@ -13,6 +13,10 @@ page_id: "api-changelog"
 
 Keep track of changes to the API for Terraform Cloud and Terraform Enterprise.
 
+### 2021-08-11
+
+* [Configuration versions](./configuration-versions.html): upload-url attribute for [uploading configuration files](./configuration-versions.html#upload-configuration-files) is only available on create response.
```

Comment:
I think there was a "Breaking change" or "Security" badge for items like these; might want to consider one of those here.